docs/sources/alerting/old-alerting/notifications.md

```diff
@@ -55,6 +55,7 @@ Name | Type | Supports images | Support alert rule tags
 [DingDing](#dingdingdingtalk) | `dingding` | yes, external only | no
 [Discord](#discord) | `discord` | yes | no
 [Email](#email) | `email` | yes | no
+[GitLab](#google-hangouts-chat) | `gitlab` | no | yes
```

Comment:
@seanarnold The anchor should be `#gitlab`.

Reply:
Thanks, will update.